### ISSUE TITLE
Better full object detection

### ISSUE THREAD

**Issue by sealedtx** [AlexeyAB/darknet#2100]
What param should i change to force network detect better full object, because i am not interested in cropped. Will just setting `jitter=0.0` do it for me? I know that this param also affects on different scales, and this is important for me. I have different size objects in my dataset and interested only in full object detection. Thank you

**Comment by AlexeyAB** [comment]
You should add to your training dataset images with: 
* labled full objects that you want to detect
* and **non-labeled cropped objects** that you don't want to detect

**Comment by sealedtx** [comment]
@AlexeyAB Thank you. What do you think about changing loss function to train better on objects that fully fit box (even with some more padding). I DON`T care about additional pixel around object if they appear, but i DO care that object should not be cropped.

**Comment by sealedtx** [comment]
@AlexeyAB basically license plates (default and square). Thats why I am not interested if number is cropped
![6028](https://user-images.githubusercontent.com/11164874/50445507-f0c74600-0917-11e9-98eb-2ddee3496fda.jpg)

**Comment by AlexeyAB** [comment]
Just label many images in a such way:
* croppped plates are un-labeled
* full plates are labeled

and Yolo will detect only Full plate (without cropped plate)

This is the only way to explain to the neural network what exactly you want to detect.

![50445507-f0c74600-0917-11e9-98eb-2ddee3496fda](https://user-images.githubusercontent.com/4096485/50447124-25dc9400-092a-11e9-947f-9d95eff7e5bd.jpg)

**Comment by AlexeyAB** [comment]
@sealedtx Yes, you can set in the each of 3 `[yolo]` layers `ignore_thresh = .8` or `0.9` instead of `.7` in your cfg-file.
But this is minor change. The main thing - you should explain to the network what you want to detect (in the training dataset).

**Comment by sealedtx** [comment]
@AlexeyAB Thanks a lot!
